src/file-attachment-element.ts

```diff
@@ -73,6 +73,17 @@ function onDragenter(event: DragEvent) {
   if (dragging) {
     clearTimeout(dragging)
   }
+
+  const dragEvent = new CustomEvent('file-attachment-dragged', {
+    bubbles: true,
+    cancelable: true,
+    detail: {
+      dragEvent: event,
+      attachmentElement: target
+    }
+  })
+  target.dispatchEvent(dragEvent)
```

Comment:
We don't check if the event was cancelled. Do we need the event to be `cancellable`?

Reply:
@koddsson that's a really good question! Even though it doesn't need to be for https://github.com/github/github/pull/190219, since this event could be used for any number of things down the road being able to `preventDefault()` on it and then checking if the event was cancelled might be useful.  I'll set `cancelable: true`

Reply:
Oh, Wednesday morning :facepalm:. It already _is_ cancellable.  Really I could go either way. I don't think we're going to regret having it be cancellable, but in truth I don't think we'll ever need to check if it's been cancelled.  Probably less is more in this case, but I'll wait to hear back.